web/client/epics/__tests__/dashboard-test.js

```diff
@@ -5,25 +5,28 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
+import axios from "../../libs/ajax";
+import MockAdapter from "axios-mock-adapter";
```

Comment:
Hi, do not mix import and require, just use import when possible or for new code